core/platform/app_identity/gae_app_identity_services_test.py

```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from core.platform.app_identity import gae_app_identity_services
+from core.tests import test_utils
+import feconf
+
+
+class GaeAppIdentityServicesTest(test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(GaeAppIdentityServicesTest, self).setUp()
+        self.application_id = test_utils.TestBase.EXPECTED_TEST_APP_ID
```

Comment:
The name should have a connotation of "expected". Perhaps self.expected_application_id or self.expected_app_id.